docs/content/preview/manage/backup-restore/point-in-time-recovery.md

```diff
@@ -170,22 +178,86 @@ If a database or a keyspace has an associated snapshot schedule, you can use tha
-| During software upgrades and downgrades, restoring back in time to the previous version is not supported | YSQL          |                                                              |
-| No support for YCQL roles and permissions.                   | YCQL          | [Issue 8453](https://github.com/yugabyte/yugabyte-db/issues/8453) |
-| No support for certain aspects of PITR in conjunction with asynchronous replication. | YSQL and YCQL | [Issue 10820](https://github.com/yugabyte/yugabyte-db/issues/10820) |
-| No support for `TRUNCATE TABLE`.                             | YSQL and YCQL | [Issue 7130](https://github.com/yugabyte/yugabyte-db/issues/7130) |
-| No support for DDL restores to a previous point in time using external backups. | YSQL and YCQL | [Issue 8847](https://github.com/yugabyte/yugabyte-db/issues/8847) |
-
-Overall development for PITR is tracked in [issue 7120](https://github.com/yugabyte/yugabyte-db/issues/7120).
\ No newline at end of file
+PITR functionality comes with several limitations, mainly related to it being used together with other features of YugabyteDB. Most of the limitations will be addressed in the future, please refer to corresponding tracking issues for more details.
+
+###  CDC
+
+Combination of PITR and [CDC](../../../explore/change-data-capture/) is currently not supported.
+
+Tracking issue: [12773](https://github.com/yugabyte/yugabyte-db/issues/12773)
+
+### xCluster replication
+
+Combination of PITR and [xCluster replication](../../../explore/multi-region-deployments/asynchronous-replication-ysql/) is not fully tested and is considered beta.
+
+xCluster does not replicate any commands related to PITR, so if you have two clusters with replication between them, you should enable PITR on both ends separately. In case of restore, the recommended procedure is the following:
+1. Stop application workloads and make sure there are no active transactions.
+1. Wait for replication to complete.
+1. Restore to the same time on both clusters.
+1. Resume the application workloads.
+
+Tracking issue: [10820](https://github.com/yugabyte/yugabyte-db/issues/10820)
+
+### Tablegroups
+
+Combination of PITR and [tablegroups](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/ysql-tablegroups.md) is currently not supported. If you attempt to create a PITR schedule within a cluster with tablegroups, you will get an error. An attempt to create a tablegroup if a schedule exists on **any** of the databases will also end up in an error.
+
+Tracking issue: [11924](https://github.com/yugabyte/yugabyte-db/issues/11924)
+
+### Global objects
+
+Global objects, such as [tablespaces](../../../explore/ysql-language-features/going-beyond-sql/tablespaces/), roles and permissions are not currently backed up by the distributed snapshots, and therefore are not supported by PITR. If you alter or drop a global object, and then try to restore to a point in time before the change, the object will **not** be restored.
```

Comment:
```suggestion
PITR doesn't support global objects, such as [tablespaces](../../../explore/ysql-language-features/going-beyond-sql/tablespaces/), roles, and permissions, because they're not currently backed up by the distributed snapshots. If you alter or drop a global object, then try to restore to a point in time before the change, the object will _not_ be restored.
```

Reply:
Question on this: will the restore fail, or will your object simply not be present in the restored state?

Reply:
It's the latter - global objects are not being tracked by PITR, therefore restore will be successful, but global objects will not be restored.